### ISSUE TITLE
Add support to Firefox 57

### ISSUE THREAD

**Issue by peylight** [downthemall/downthemall-legacy#269]
### Your DownThemAll! and Browser Version and Language and Operating System
Firefox 57 Developer Edition
English
Windows 10

### Expected Behavior
Support this version

### Actual Behavior
Not support this version

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*

**Comment by Xmetalfanx** [comment]
As seen on https://github.com/downthemall/downthemall/issues/256

http://www.downthemall.net/re-downthemall-and-webextensions-or-why-why-i-am-done-with-mozilla/ 

THOUGH better news http://www.downthemall.net/progress/

**Comment by regs01** [comment]
Making WebExtension for Chrome and Edge would be a good lesson for mozilla

**Comment by josephj11** [comment]
I want it for Vivaldi (based on Chromium)!

Firefox has consistently broken features I loved ever since version 4. Vivaldi is the opposite!
Sometimes when they add a new feature I wanted, it's actually better than what I wanted.

Right now, I can wait a little bit for DTA because Flashgot, which I use to trigger DTA, is broken too!

Flashgot/DTA is almost the only reason I still use Firefox. That combination is light years ahead of any similar software that I'm aware of.

**Comment by Timi7007** [comment]
Just to keep this updated: http://www.downthemall.net/delays/

**Comment by EMBBlaster** [comment]
Should be closed as duplicate of issue #240.

**Comment by rhadamanthe** [comment]
Hi.

Does anyone know where is located the source code for DTA lite? I have not seen any branch here.
I have started to work on a [web extension](https://github.com/rhadamanthe/host-grabber-pp) in replacement of [Image Host Grabber](https://addons.mozilla.org/fr/firefox/addon/imagehost-grabber/). And Firefox's new download API is so limited that I would like to give a hand on DTA. The idea would be to make a greater download API (with resume, queue persistence, etc) that other web extensions might use too.

**Comment by bperel** [comment]
I would be interested to know where the source code of DTA lite is as well, even if it isn't stable yet

**Comment by xdhmoore** [comment]
I would as well.

**Comment by Xmetalfanx** [comment]
I think (I have no idea the roadmap for this but it is on Github) using Waterfox for now if you can is the way to go with the original extension.    

I may have said it already and forgot but (@ the original dev) I think that so many people appreciate and love the project that even if you "no longer want" to develop the extension, there may be talented people out there that could (even if 100% incomplete ATM with DTA lite) pick up the project and maybe get your "blessing" to "take over" DTA where its not "all on you". ... or at least where you are 100% still in "charge" of DTA but they can contribute with PR's and such to chip-in on the effort.

**Comment by MartinX3** [comment]
Basilisk Browser (Palemoon on Firefox v52 Base) is a solution.

**Comment by g-i-o-r-g-i-o** [comment]
I hope that this comes soon...

**Comment by g-i-o-r-g-i-o** [comment]
...

**Comment by Xmetalfanx** [comment]
I hate to get off topic but even if its a good project ... I start getting iffy about using a fork of a fork of a fork type of things ... I mean what is the advantage of say "Pale Moon on an updated Firefox base" (compared to ACTUAL Pale Moon" vs something like Waterfox which IIRC is a fork of FF and you eliminate a "middle man fork" so to speak. 

AND OF COURSE I realize I am saying this on github lol

**Comment by Tobias004** [comment]
DTH is DEAD.  Just look at the commits, or wait forever!

**Comment by Anutrix** [comment]
Why won't the developer reply or post something on his website?

**Comment by josephj11** [comment]
@Anutrix I guess no point in beating a dead horse. The way Firefox/Mozilla has treated extension authors got many of them demoralized enough to quit long before quantum nuked almost all of them.

**Comment by Xmetalfanx** [comment]
they were giving warnings for months (that doesn't mean it's easy for some addons to be completely redone ... look HOW DIFFERENT Noscript is now) .... I have just expected this is a dead project and the dev has moved on ... would expect a "if someone wants to officially take over the project" message but .. whatever

**Comment by MartinX3** [comment]
Yes. A missing communication is very very very impolite.

**Comment by xdhmoore** [comment]
I agree it would be nice to hear something, but in the end, it's a bunch of free work someone did as a gift to the community. I don't think they are under any obligation to follow up if they don't have time any more.
